### PR TITLE
check the number of CPU in Mac OS.

### PR DESCRIPTION
**Pull request by mori0711**
I added the cord that checks the number of CPU in Mac OS.

### CODE REVIEW

File: src/burai/com/env/MacCPUInfo.java [hunk 1]
```diff
@@ -22,6 +30,32 @@ protected int countNumCPUs() {
          *  TODO: get #cpus for MacOS.
          */
 
-        return 1;
+        int numCPUs = 0;
+
+        try {
+            Process process = Runtime.getRuntime().exec(CHK_MAC_CPUCOM);
```
Comment (nisihara1) [inline src/burai/com/env/MacCPUInfo.java]: It is better to wait for the termination of the process, and to check exit code.
If exit code is not zero, numCPUs = 1 and return this method.

File: src/burai/com/env/MacCPUInfo.java [hunk 2]
```diff
@@ -22,6 +30,32 @@ protected int countNumCPUs() {
...
+            BufferedReader in = new BufferedReader(new InputStreamReader(process
+                .getInputStream()));
+
+            String line = null;
+            while ((line = in.readLine()) != null) {
+                line = line.trim();
+                if (line.contains(CPU_WORD)){
+                    String[] nCPULine = line.split(" ", 0);
+                    if (nCPULine.length == 0){
+                        break;
```
Comment (nisihara1) [inline src/burai/com/env/MacCPUInfo.java]: set numCPUs = 1

File: src/burai/com/env/MacCPUInfo.java [hunk 3]
```diff
@@ -22,6 +30,32 @@ protected int countNumCPUs() {
...
+                    }
+                    else {
+                        numCPUs = Integer.parseInt(nCPULine[nCPULine.length -1]);
+                    }
+                }
+            }
+
+        } catch (IOException e) {
+            // TODO
+            e.printStackTrace();
```
Comment (nisihara1) [inline src/burai/com/env/MacCPUInfo.java]: set numCPUs = 1

File: src/burai/com/env/MacCPUInfo.java [hunk 4]
```diff
@@ -22,6 +30,32 @@ protected int countNumCPUs() {
...
+            String line = null;
+            while ((line = in.readLine()) != null) {
+                line = line.trim();
+                if (line.contains(CPU_WORD)){
+                    String[] nCPULine = line.split(" ", 0);
+                    if (nCPULine.length == 0){
+                        break;
+                    }
+                    else {
+                        numCPUs = Integer.parseInt(nCPULine[nCPULine.length -1]);
```
Comment (nisihara1) [inline src/burai/com/env/MacCPUInfo.java]: catch NullPointerException and NumberFormatException

File: src/burai/com/env/MacCPUInfo.java (hunk 1 above)
Comment (mori0711) [inline src/burai/com/env/MacCPUInfo.java]: OK

File: src/burai/com/env/MacCPUInfo.java (hunk 3 above)
Comment (mori0711) [inline src/burai/com/env/MacCPUInfo.java]: OK

**Pull request closed without merging**